client/gradle/openapi.gradle

```diff
@@ -18,11 +17,12 @@ swaggerSources {
               "--group-id", "${artifactGroup}",
               "--artifact-version", "${version}",
               "--ignore-file-override", "${projectDir}/.swagger-codegen-ignore",
-              "--additional-properties", "errorOnUnknownEnum=true",
+              "--additional-properties", "errorOnUnknownEnum=false",
```

Comment:
The later versions of swagger codegen have an error with this is true. I tried to update the template locally but could not get it to work. I suspect no one in the real world uses this or it would have been fixed already.